### DISCUSSION TITLE
Send email with Vlan change

### DISCUSSION THREAD

**Discussion by tomaszcwik1** [netbox-community/netbox#15331] [Help Wanted!]
Hi Everyone !
I wonder if it is possible to use the "Event Rules" to send mail after made some change
i see similar topic on YT https://www.youtube.com/watch?v=eavpywH0LfY but i cannot find way to send mail notification.
Any ideas how to do this ? :D

**Comment by zmaster7** [comment]
For sending email via Event Rules, you could set the action to a custom script and utilize the send_mail function with django: [https://docs.djangoproject.com/en/4.2/topics/email/](https://docs.djangoproject.com/en/4.2/topics/email/).
